nodepool/files/nodepool.yml

```diff
@@ -38,6 +38,8 @@ providers:
     cloud: public_cloud
     boot-timeout: 600
     hostname-format: nodepool-{label.name}-{provider.name}-{node.id}
+    # Ignore the quota given by public cloud and only respect what we provide as max-servers
+    ignore-provider-quota: True
```

Comment:
``ignore-provider-quota`` is a ``pool`` parameter, not a ``provider`` parameter, so this needs to go into each pool definition